releasenotes/notes/setter-methods-for-experiment-099074e59faffb49.yaml

```diff
@@ -0,0 +1,17 @@
+---
+features:
+  - |
+    Added ``experiment_type`` as optional ``__init__`` kwarg in BatchExperiment
```

Comment:
There are two separate things in this PR: the setter and the args in init. You can separate them in the release notes.